diff --git a/node_modules/eslint-plugin-prettier/worker.mjs b/node_modules/eslint-plugin-prettier/worker.mjs
index b76274c..e023812 100644
--- a/node_modules/eslint-plugin-prettier/worker.mjs
+++ b/node_modules/eslint-plugin-prettier/worker.mjs
@@ -45,9 +45,7 @@ runAsWorker(
         resolveConfig: false,
         withNodeModules: false,
         ignorePath: '.prettierignore',
-        plugins: /** @type {string[] | undefined} */ (
-          prettierRcOptions ? prettierRcOptions.plugins : undefined
-        ),
+        ...prettierRcOptions,
         ...eslintFileInfoOptions,
       },
     );
